docs/metadata.md

```diff
@@ -36,7 +36,6 @@ the workload template with an init container section that looks like:
         command: ["python3", "stockpile-wrapper.py"]
         args:
           - -s={{ elasticsearch.server }}
-          - -p={{ elasticsearch.port }}
```

Comment:
Elastic search.server needs to be .url ^

Reply:
fixed!